packages/wdio-cli/src/constants.ts

```diff
@@ -186,13 +186,13 @@ export const QUESTIONNAIRE = [{
     type: 'input',
     name: 'hostname',
     message: 'What is the host address of that cloud service?',
-    when: /* istanbul ignore next */ (answers: Questionnair) => answers.backend.toString().indexOf('different service') > -1
+    when: /* istanbul ignore next */ (answers: Questionnair) => String(answers.backend).indexOf('different service') > -1
```

Comment:
Can we do this instead everywhere? This seems to be a type issue and if we just case the type the compiled code stays the same.

```suggestion
    when: /* istanbul ignore next */ (answers: Questionnair) => (answers.backend as string).indexOf('different service') > -1
```

Reply:
Sure done!